app/src/main/res/layout/fragment_move.xml

```diff
@@ -69,48 +69,87 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
-        <androidx.core.widget.NestedScrollView
+        <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:animateLayoutChanges="true"
-            android:background="@color/backgroundColor"
-            android:clipToPadding="false"
-            android:paddingTop="@dimen/alternativeMargin"
-            android:paddingBottom="@dimen/marginStandardMedium"
-            android:scrollbars="none">
+            android:orientation="vertical">
 
-            <LinearLayout
+            <com.google.android.material.textfield.TextInputLayout
+                android:id="@+id/searchInputLayout"
+                style="@style/SearchTextInputLayoutStyle"
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/searchViewHeight"
+                android:layout_marginStart="@dimen/marginStandardSmall"
+                android:layout_marginEnd="@dimen/marginStandardMedium"
```

Comment:
Why have a margin start and end different from one another?

Reply:
That's because of the endIcon of the search view, we cannot set endIcon padding. The same logic is already used in the search item. At least the margin doesn't need to add a tools:disableRTLSymetry compared to setting an end padding alone